pkgs/by-name/cr/cruft/package.nix

```diff
@@ -1,34 +1,35 @@
 {
   python3Packages,
-  fetchFromGitHub,
+  fetchPypi,
   lib,
 }:
 python3Packages.buildPythonApplication rec {
   pname = "cruft";
-  version = "2.15.0";
+  version = "2.16.0";
   pyproject = true;
 
-  src = fetchFromGitHub {
-    owner = "cruft";
-    repo = "cruft";
-    rev = version;
-    hash = "sha256-qIVyNMoI3LsoOV/6XPa60Y1vTRvkezesF7wF9WVSLGk=";
+  src = fetchPypi {
```

Comment:
Please keep using `fetchFromGitHub` if possible.

Reply:
Could you link some issue where this is discussed? Upstream might do some editional things before publishing, like in this case, changing version (which generates `cruft-0.0.0.dist-info` in $out here)

Anyway, changed to `fetchFromGitHub`.